docs/settings.rst

```diff
@@ -0,0 +1,22 @@
+Settings
+========
+
+Settings can be configured via the ``HEALTH_CHECK`` dictionary.
+
+``psutil``
+----------
+
+The following default settings will be used to check for disk and memory
+utilization. If you would prefer different thresholds, you can add the dictionary
+below to your Django settings file and adjust the values accordingly. If you want
+to disable any of these checks, set its value to ``None``.
+
+.. code:: python
+
+    HEALTH_CHECK = {
+        'DISK_USAGE_MAX': 90,  # percent
```

Comment:
In this section it would be good to actually have a headline for each of those keys in the dictionary and briefly explain what they are for.